wlm_server/lock/models.py

```diff
@@ -0,0 +1,26 @@
+from datetime import datetime
+
+from django.db import models
+
+from user.models import User
+from channel.models import Channel
+from config.models import Config
+
+def get_default_expires_at():
+    config = Config.objects.first()
```

Comment:
Is this the latest config record?

Reply:
As I said in PR explanation, it is forced that there is only one record in `config` model.
This constraint was implemented through `models.UniqueConstraint` in `config/models.py`.

Reply:
There might be more efficient way to store the global config, but I have initially adopted the simplest approach.

Reply:
Oh, sorry I missed it, now I see! Thank you for the explanation.